src/main/java/com/box/sdk/BoxJSONObject.java

```diff
@@ -181,7 +186,7 @@ void removePendingChange(String key) {
     }
 
     /**
-     * Updates this BoxJSONObject using the information in a JSON object.
+     * Updates this BoxJSONObject using the information in a JSON object and preserves the JSON object.
      * @param jsonObject the JSON object containing updated information.
      */
     void update(JsonObject jsonObject) {
```

Comment:
```suggestion
    void update(JsonObject jsonObject) {
        this.jsonObject = jsonObject;
        
```

Reply:
I think if we move line 200 to here, then it's slightly more beneficial if the parseJSONMember call fails due to parsing exception.  At least in that case, the raw JSON is preserved.